### PR TITLE
Added functionality to register user

### PR DESCRIPTION
**Pull request by tcozzens**
You can see that the key is in there, we may want to move this out at some point. 

oh purple peacock

### CODE REVIEW

File: users/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "deploy": "npm run clean & npm run predeploy & npm run zip & npm run publish",
     "publish": "aws2 --profile confyeti lambda update-function-code --function-name confyeti-api-users --zip-file fileb://confyeti-api-users.zip",
     "test": "mocha 'tests/*.test.js'",
-    "zip": "7z a confyeti-api-users.zip index.js.tmp & 7z rn confyeti-api-users.zip index.js.tmp index.js"
+    "zip": "7z a confyeti-api-users.zip index.js.tmp & 7z a confyeti-api-users.zip config.json & 7z rn confyeti-api-users.zip index.js.tmp index.js"
```
Comment (tcozzens) [inline users/package.json]: @mikejancar  went with what you said here. I also tried it with adding ```config.json``` after that ```index.js.temp``` and it did work - not sure if it was a fluke lol

File: users/index.js [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+
+  await callSignUp(params);
+
+  return sharedFunc.createResponse(200, { successMessage: 'User registered, email sent' });
+}
+
+async function callSignUp(params) {
+  const cognitoidentityserviceprovider = new AWS.CognitoIdentityServiceProvider();
+
+  console.log('test');
```
Comment (mikejancar) [inline users/index.js]: Just a reminder to clean this up on the next one.

File: users/package.json (hunk 1 above)
Comment (mikejancar) [reply users/package.json]: As long as it works...lol

Comment (tcozzens) [reply users/package.json]: left it as you said

File: users/index.js (hunk 2 above)
Comment (tcozzens) [reply users/index.js]: done

**Comment by tcozzens** [comment]
[Research how to handle user logins securely](https://trello.com/c/5cY0iRPV/25-research-how-to-handle-user-logins-securely)

**Pull request merged**